apps/flipper_auth/pubspec.yaml

```
@@ -25,21 +23,24 @@ dependencies:
   # url_launcher: ^6.3.2
   json_annotation: ^4.9.0
-  mobile_scanner: ^7.0.0-beta.8
+  mobile_scanner: ^7.0.1
   flipper_scanner: any
   flipper_models: any
   flipper_services: any
   sqlite3_flutter_libs: ^0.5.39
   realm: ^20.1.1
   pubnub: ^5.2.0
   connectivity_plus: ^6.0.5
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)